Sources/Support/StoreMessageType.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreMessageType.swift
+//
+//  Created by Antonio Rico Diez on 27/9/23.
+
+import StoreKit
+
+/// Types of messages available in StoreKit
```

Comment:
Nit: an enum, represents one type
```suggestion
/// Type of messages available in StoreKit
```